megamek/src/megamek/client/ui/swing/CommonMenuBar.java

```diff
@@ -217,606 +134,294 @@ public CommonMenuBar(Client parent) {
-        /* TODO: moveTraitor = createMenuItem(
-                menu,
-                Messages.getString("CommonMenuBar.moveTraitor"), MovementDisplay.MOVE_TRAITOR); //$NON-NLS-1$ */
-
-        //create aero movement sub-menu
-        aeromenu = new JMenu(Messages.getString("CommonMenuBar.AeroMenu")); //$NON-NLS-1$
-        moveAcc = createMenuItem(aeromenu, Messages.getString("CommonMenuBar.moveAcc"), MovementDisplay.MoveCommand.MOVE_ACC.getCmd()); //$NON-NLS-1$
-        moveDec = createMenuItem(aeromenu, Messages.getString("CommonMenuBar.moveDec"), MovementDisplay.MoveCommand.MOVE_DEC.getCmd()); //$NON-NLS-1$
-        moveAccN = createMenuItem(aeromenu, Messages.getString("CommonMenuBar.moveAccN"), MovementDisplay.MoveCommand.MOVE_ACCN.getCmd()); //$NON-NLS-1$
-        moveDecN = createMenuItem(aeromenu, Messages.getString("CommonMenuBar.moveDecN"), MovementDisplay.MoveCommand.MOVE_DECN.getCmd()); //$NON-NLS-1$
-        moveEvadeAero = createMenuItem(aeromenu, Messages.getString("CommonMenuBar.moveEvadeAero"), MovementDisplay.MoveCommand.MOVE_EVADE_AERO.getCmd()); //$NON-NLS-1$
-        moveRoll = createMenuItem(aeromenu, Messages.getString("CommonMenuBar.moveRoll"), MovementDisplay.MoveCommand.MOVE_ROLL.getCmd()); //$NON-NLS-1$
+        createMenuItem(menu, getString("moveWalk"), MOVE_WALK.getCmd(), KeyEvent.VK_W); 
+        createMenuItem(menu, getString("moveJump"), MOVE_JUMP.getCmd(), KeyEvent.VK_J); 
+        createMenuItem(menu, getString("moveSwim"), MOVE_SWIM.getCmd(), KeyEvent.VK_S); 
+        createMenuItem(menu, getString("moveModeConvert"), MOVE_MODE_CONVERT.getCmd(), KeyEvent.VK_C); 
+        createMenuItem(menu, getString("moveBackUp"), MOVE_BACK_UP.getCmd()); 
+        createMenuItem(menu, getString("moveGetUp"), MOVE_GET_UP.getCmd()); 
+        createMenuItem(menu, getString("moveGoProne"), MOVE_GO_PRONE.getCmd()); 
+        createMenuItem(menu, getString("moveTurn"), MOVE_TURN.getCmd()); 
+        createMenuItem(menu, getString("moveNext"), MOVE_NEXT.getCmd(), KeyEvent.VK_N); 
+        createMenuItem(menu, getString("moveForwardIni"), MOVE_FORWARD_INI.getCmd()); 
+        createMenuItem(menu, getString("moveRaise"), MOVE_RAISE_ELEVATION.getCmd()); 
+        createMenuItem(menu, getString("moveLower"), MOVE_LOWER_ELEVATION.getCmd()); 
+        createMenuItem(menu, getString("moveReckless"), MOVE_RECKLESS.getCmd()); 
+        createMenuItem(menu, getString("moveEvade"), MOVE_EVADE.getCmd()); 
+        createMenuItem(menu, getString("moveBootlegger"), MOVE_BOOTLEGGER.getCmd()); 
+        createMenuItem(menu, getString("moveShutdown"), MOVE_SHUTDOWN.getCmd()); 
+        createMenuItem(menu, getString("moveStartup"), MOVE_STARTUP.getCmd()); 
+        createMenuItem(menu, getString("moveSelfDestruct"), MOVE_SELF_DESTRUCT.getCmd()); 
+        /* TODO: moveTraitor = createMenuItem(menu, getString("moveTraitor"), MovementDisplay.MOVE_TRAITOR);  */
```

Comment:
Why keep this?

Reply:
Traitoring is currently only present in the context menu in the movement phase which is a bit silly. I can see it put here instead.